### PR TITLE
Remove Android MediaPlayer support #2656

### PR DESCRIPTION
**Pull request by freeboub**
#### Update the documentation
Remove MediaPlayer References (execpt in old release description)

#### Provide an example of how to test the change
Tested in basic sample (path integration updated)

#### Focus the PR on only one area
Just ensure changing the exoplayer-android to exoplayer is enough

#### Describe the changes
remove deprecated Media Player implementation as requested here:
https://github.com/react-native-video/react-native-video/issues/2656

### CODE REVIEW

**Comment by evanjmg** [comment]
Plz merge this

**Comment by michaelbayday** [comment]
Could you folks please merge this?

**Comment by hueniverse** [comment]
Everyone - Please stop asking for things to be merged. Bugging the maintainers is not going to magically create free resources and time. We are doing the best we can with the resources we have. Nothing is preventing you from forking and creating temporary patched versions until the main repo is back to normal. This will take a few weeks to a few months. Thank you for your understanding.

**Comment by hueniverse** [comment]
@armadilio3 Is this PR going to create merge issues for your Android PR coming?

**Comment by freeboub** [comment]
@hueniverse i wait your GO to fix conflicts ;) thanks !

**Comment by armands-malejevs** [comment]
@hueniverse I think that there is a good chance that it will. Looking at the changes it appears that some Exoplayer files have been moved, really depends on how well git will resolve moving files and editing them.

I would really want to see this merged in, but I'd hold off until we merge #2689 and deal with the conflicts after.

**Comment by hueniverse** [comment]
@freeboub If you can help review #2689 that would move things along!

**Comment by freeboub** [comment]
I did a git mv on folder so it should work but it seems it doesn't work :'(
Can can re create the patch if needed, it was 1 hour job :)
I agree let's merge @armadilio3 PR first !

**Comment by freeboub** [comment]
Thinking back on this PR, @hueniverse don't you think it would be better to swap players ? Ie, put exoplayer as default and keep old mediaplayer for 'dummy' video use cases ?

**Comment by evanjmg** [comment]
Doesn’t using the old player require jcenter? Best to remove altogether

**Comment by hueniverse** [comment]
@freeboub less code is better. We don't have enough resources so we should support the minimum possible.

File: README.md
Comment (armands-malejevs) [inline README.md:338]: Can we just use `Android` instead of `Android ExoPlayer` since we only have ExoPlayer

File: android/build.gradle
Comment (armands-malejevs) [inline android/build.gradle:12]: Is this required? We might run into issues on older Android devices that only support Java 7 features, since the issue will no longer be caught by the compiler. This change itself shouldn't cause issues, but if a developer uses Java 8 features then it might not work on Android 7 and older devices.

**Review by armands-malejevs — APPROVED** [review]
Looks good. See some of my minor comments if you want to address them or not.

File: android/build.gradle
Comment (freeboub) [reply android/build.gradle:12]: I am not sure, but I think yes ... This is not my code, it already there in master branch :/ I just did a " git mv "

Comment (freeboub) [reply android/build.gradle:12]: It was introduced in version : 5.1.0-alpha7

**Comment by freeboub** [comment]
I am not able to merge this PR with master, it was easier to create a new one here:
https://github.com/react-native-video/react-native-video/pull/2724

Regarding to comments:
- I fix Readme.md to change Android Exoplayer to Android (and remove Android MediaPlayer of course)
- For JAVA 1.8, I don't think it will cause interoperability issues:  https://developer.android.com/studio/write/java8-support

**Pull request closed without merging**
